myocamlbuild.ml.in

```diff
@@ -16,9 +16,9 @@ let piqic_rule () : unit =
            A (expand "${piqic}");
            A (expand "${piqic_flags}");
            A "-C";
-           A "lib/bap_types";
+           A "../_build/piqi";
```

Comment:
I didn't know the little language or variable incantation within the
quotation in order to be able to write it relative to the project root.
On Oct 23, 2015 6:31 AM, "Ivan Gotovchits" <notifications@github.com> wrote:

> In myocamlbuild.ml.in
> <https://github.com/BinaryAnalysisPlatform/bap/pull/307#discussion_r42851694>
> :
>
> > @@ -16,9 +16,9 @@ let piqic_rule () : unit =
> >             A (expand "${piqic}");
> >             A (expand "${piqic_flags}");
> >             A "-C";
> > -           A "lib/bap_types";
> > +           A "../_build/piqi";
>
> hwe shouldn't harcode _build here, is it required?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BinaryAnalysisPlatform/bap/pull/307/files#r42851694>.
>